tests/test_io.py

```diff
@@ -969,7 +971,7 @@ def _get_diagnostics_cancelled_sim(self, result_file_name):
         model.initialize()
 
         # Need to mock the diagnostics variables in order to invoke simulation_start
-        diagnostics_start_name = "Diagnostics."
+        diagnostics_start_name = diagnostics_prefix
```

Comment:
I don't think diagnostics_start_name is needed anymore. Fix if you like.

Reply:
Good point! I've fixed it now